Lib/fontTools/subset/__init__.py

```diff
@@ -1388,6 +1388,9 @@ def subset_glyphs(self, s):
 def subset_glyphs(self, s):
   self.strikeData = [{g: strike[g] for g in s.glyphs if g in strike}
 					 for strike in self.strikeData]
+  # Prune empty strikes
+  # https://github.com/fonttools/fonttools/issues/1633
+  self.strikeData = list(filter(bool, self.strikeData))
```

Comment:
this works, but the canonical way for pruning empty/null items from a sequence is passing `None` as filter

```suggestion
  self.strikeData = list(filter(None, self.strikeData))
```

https://docs.python.org/3/library/functions.html#filter

> If function is None, the identity function is assumed, that is, all elements of iterable that are false are removed.

A list comprehension would be equally fine:

```python
self.strikeData = [s for s in self.strikeData if s]
```